build/common.sh

```diff
@@ -97,27 +97,31 @@ kube::build::get_docker_wrapped_binaries() {
           kube-apiserver,busybox
           kube-controller-manager,busybox
           kube-scheduler,busybox
+          federated-apiserver,busybox
```

Comment:
Not sure what order pattern this is using, but I think it should either be alphabetical or grouped. I am inclined towards the latter, probably with a comment before the group saying that these are federation binaries. Example:

```shell
# Cluster federation binaries
federated-apiserver,busybox
```